vignettes/cookbook_preparatory_steps.Rmd

```diff
@@ -18,106 +18,108 @@ library(pacta.loanbook)
 
-For sectors with technology level trajectories, the data set provides the TMSR and SMSP pathways based on the Market Share Approach, an allocation rule that implies all companies active in a sector have to adjust their production in a way that keeps market shares constant and solves for the aggregate climate transition scenario. For more information on how to calculate the TMSR and the SMSP, see the [PACTA for Banks documentation](https://rmi-pacta.github.io/r2dii.analysis/articles/target-market-share.html).
+A more detailed explanation of the market share approach can be found [here](cookbook_metrics.html).
 
-The target market share scenario data set must be a CSV file and contains the following columns:
+The Target Market Share Scenario dataset must be a CSV file and include the following columns:
 
 ```{r cols_tms_scenario, echo = FALSE, results = 'asis'}
 pacta.loanbook:::list_col_names_and_types(scenario_demo_2020)
 ```
 
-For more detail about the necessary structure of this dataset, see the data dictionary for [scenario](data_scenario.html#climate-transition-scenario-dataset-technology-pathways-for-target-market-share-calculation).
+For details on the required structure of this dataset, refer to the [market share scenario data dictionary](data_scenario.html#climate-transition-scenario-dataset-technology-pathways-for-target-market-share-calculation).
 
-For sectors that do not have technology level pathways, PACTA uses the Sectoral Decarbonization Approach (SDA), an allocation rule that implies that all companies in a sector have to converge their physical emission intensity at a future scenario value - e.g. in the year 2050. This implies that more polluting companies have to reduce their physical emissions intensity more drastically than companies using cleaner technology. It does not have any direct implications on the amount of units produced by any company. For further information on calculating the SDA in PACTA, please see the [PACTA for Banks documentation](https://rmi-pacta.github.io/r2dii.analysis/articles/target-sda.html).
+For sectors without technology-level pathways, PACTA applies the Sectoral Decarbonization Approach (SDA). This method requires all companies in a sector to align their physical emission intensity with a future scenario target (e.g., by 2050). Higher-emission companies must reduce their intensity more drastically than those already using cleaner technologies. However, SDA does not directly affect production volumes.
 
-The SDA scenario data set must be a CSV file and contains the following columns:
+A more detailed explanation of the sectoral decarbonization approach can be found [here](cookbook_metrics.html)
+
+The SDA scenario dataset must be a `.csv` file that contains the following columns:
```

Comment:
```suggestion
The SDA Scenario dataset must be a `.csv` file that contains the following columns:
```
to keep spelling consistent between this file and the market share scenario file, this should be capitalized

Reply:
Hmmm, I've added this as something to consider in: https://github.com/RMI-PACTA/pacta.loanbook/issues/141

In general, we are a a bit all over the place with capitalization... but "scenario" is just a word, not necessarily a proper noun, so I don't think it should be capitalized?

Reply:
I would generally rather go in the opposite direction, and sweep the cookbook to make the word lowercase in all instances.